pkgs/development/python-modules/gios/default.nix

```diff
@@ -0,0 +1,57 @@
+{ lib
+, aiohttp
+, aioresponses
+, buildPythonPackage
+, click
+, fetchFromGitHub
+, pytest-asyncio
+, pytest-error-for-skips
+, pytestCheckHook
+, pythonOlder
+}:
+
+buildPythonPackage rec {
+  pname = "gios";
+  version = "1.0.1";
+  disabled = pythonOlder "3.7";
+
+  src = fetchFromGitHub {
+    owner = "bieniu";
+    repo = pname;
+    rev = version;
+    sha256 = "031z5j1j961pcw8s4ajj5z2xn2jxb2hyfn3x3bxbaagwl9a5qkvn";
+  };
+
+  propagatedBuildInputs = [
+    aiohttp
+    click
```

Comment:
Hmm, why the click dependency? I don't see any references to it in the repo.

Reply:
I guess that it's a left-over. Removed, thanks for the feedback.